### ISSUE TITLE
Keyboard shortcuts does not work with different keyboard layouts

### ISSUE THREAD

**Issue by HKalbasi** [vector-im/element-web#17220]
When I use Farsi keyboard layout, keyboard shortcuts like `Ctrl+K` (= `Ctrl + ن` in Farsi) does not work. Which should work I think. I tested in latest element web.

**Comment by t3chguy** [comment]
Closing in favour of https://github.com/vector-im/element-web/issues/21067
